service-tests/github.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'github', title: 'Github' });
+module.exports = t;
+
+
+t.create('File size')
+  .get('/size/webcaetano/craft/build/craft.min.js.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('size'),
+    value: Joi.string().regex(/^[0-9]*[.]?[0-9]+\s(B|kB|MB|GB|TB|PB|EB|ZB|YB)$/)
+  }));
```

Comment:
❤️ 

Is it possible to hit both the `repo or file not found` and `unknown file` branches? Could you test these as well?